assets/model_monitoring/components/src/data_quality_compute_metrics/compute_data_quality_metrics.py

```diff
@@ -539,6 +539,17 @@ def compute_data_quality_metrics(df, data_stats_table):
         when(col("dataType") == "DoubleType()", "Numerical")
         .when(col("dataType") == "LongType()", "Numerical")
         .when(col("dataType") == "StringType()", "Categorical")
+        .when(col("dataType") == "IntegerType()", "Numerical")
+        .when(col("dataType") == "TimestampType()", "Categorical")
+        .when(col("dataType") == "BooleanType()", "Categorical")
+        .when(col("dataType") == "BinaryType()", "Categorical")
```

Comment:
can we have some tests with these types

Reply:
Yes, I have unit test for all supported and some unsupported type. The data quality metrics are generated as we expected. no failure or exception for unsupported type. Also we have a e2e test to cover common datatypes.